app/kumactl/cmd/manage/ca/provided.go

```diff
@@ -0,0 +1,176 @@
+package ca
+
+import (
+	kumactl_cmd "github.com/Kong/kuma/app/kumactl/pkg/cmd"
+	"github.com/Kong/kuma/app/kumactl/pkg/output/printers"
+	"github.com/Kong/kuma/pkg/core/ca/provided/rest/types"
+	"github.com/Kong/kuma/pkg/tls"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+)
+
+func newProvidedCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "provided",
+		Short: "Manage certificate provided authorities",
+		Long:  `Manage certificate provided authorities.`,
+	}
+	// sub-commands
+	cmd.AddCommand(newCertificatesCmd(pctx))
+	cmd.AddCommand(newDeleteCaCmd(pctx))
+	return cmd
+}
+
+func newDeleteCaCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete",
```

Comment:
What is the use case for `kumactl manage ca provided delete` ?
Why `kumactl manage ca provided certificates delete` is not enough ?